packages/ra-supabase-core/src/useSupabaseAccessToken.test.tsx

```diff
@@ -86,7 +106,7 @@ describe.skip('useSupabaseAccessToken', () => {
         });
     });
 
-    test('should not redirect users if the access token is not present in the URL and redirectTo is false', async () => {
+    test.skip('should not redirect users if the access token is not present in the URL and redirectTo is false', async () => {
```

Comment:
Why?

Reply:
All your test for useSupabaseAccessToken were skip. So i just unskip the ones i managed to get working

Reply:
:scream: Let's re-enable them then. That's most probably a mistake

Reply:
Just noticed the `describe` comment, let's leave it that way for now